src/renderers/dom/shared/ReactBrowserEventEmitter.js

```diff
@@ -171,6 +171,25 @@ var ReactBrowserEventEmitter = Object.assign({}, ReactEventEmitterMixin, {
             'scroll',
             mountAt,
           );
+        } else if (
+          dependency === 'topMouseEnter' ||
+          dependency === 'topMouseLeave'
+        ) {
+          if (isEventSupported('mouseenter', true)) {
```

Comment:
If we're assuming all supported browsers will support `mouseenter` events, do we need the `isEventSupported` check?